rs/nervous_system/integration_tests/tests/advance_sns_target_version.rs

```diff
@@ -0,0 +1,255 @@
+use ic_nervous_system_integration_tests::pocket_ic_helpers::{await_with_timeout, sns};
+use ic_nervous_system_integration_tests::{
+    create_service_nervous_system_builder::CreateServiceNervousSystemBuilder,
+    pocket_ic_helpers::{
+        add_wasm_via_nns_proposal, add_wasms_to_sns_wasm, install_nns_canisters, nns,
+    },
+};
+use ic_nns_test_utils::sns_wasm::create_modified_sns_wasm;
+use ic_sns_governance::{
+    governance::UPGRADE_STEPS_INTERVAL_REFRESH_BACKOFF_SECONDS,
+    pb::v1 as sns_pb,
+    pb::v1::upgrade_journal_entry::{Event, TargetVersionSet, UpgradeStepsRefreshed},
+};
+use ic_sns_swap::pb::v1::Lifecycle;
+use ic_sns_wasm::pb::v1::SnsCanisterType;
+use pocket_ic::PocketIcBuilder;
+
+#[tokio::test]
+async fn test_get_upgrade_journal() {
```

Comment:
I dont think theres much benefit to having a duplicated `test_get_upgrade_journal` here, can we just use `test_advance_target_version_upgrades_all_canisters`?